### PR TITLE
Rework how we generate chef repos a bit

### PR DESCRIPTION
**Pull request by tas50**
I think this is going to need some further refinement, but the technical aspect of it is correct now. The command is correct and the "manual" bit is all gone since this is not manual.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: chef_master/source/install_chef_air_gap.rst [hunk 1]
```diff
@@ -148,13 +148,13 @@ Install Chef DK
 
    .. code-block:: bash
 
-      dpkg -i chefdk_2.0.28-1_amd64.deb
+      dpkg -i chefdk_3.2.30-1_amd64.deb
```
Comment (kagarmoe) [inline chef_master/source/install_chef_air_gap.rst:151]: Will this DK version work with A1?

Comment (tas50) [reply chef_master/source/install_chef_air_gap.rst:151]: Shouldn't be an issue. Also if we tell them to go to downloads they're going to get a version similar to this. Mostly I just try to update example chef or DK versions when I see them. Docs that mention apps versions from years ago tells me the docs are potentially outdated as well.

**Comment by tas50** [comment]
LGTM. Thanks for the fixes @kagarmoe

**Comment by mjingle** [comment]
FYI @tas50 - Headers can't jump from H1 to H3 in these Sphinx documents. You can have all the H1 headers you want, but there must be a H1 -> H2 -> H3 progression with no jumps.

**Review by mjingle — APPROVED** [review]
tenor-76947173

**Pull request merged**